app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1> Project Description </h1>
+
+<li> Name: <%= @project.name %></li>
+<li>Material: <%= @project.material %></li>
+<li>Challenge Theme: <%= @project.challenge.theme %></li>
+<li>Number of Contestants: <%= @project.contestants.count%></li>
```

Comment:
This would be considered an MVC infraction, since `.count` is a calculation method, and we want to only display _tested_ calculations to our users in the view. So, I'd move this to a model method and then write a model test ensuring the calculation works, then call this method in the view.